packages/pages/src/upgrade/pages_updater.ts

```diff
@@ -7,15 +6,19 @@ const replacementText = '"@yext/sites-components"';
 
 // Function to install npm packages
 const installPackages = (targetDirectory: string) => {
+  const packagePath = path.resolve(targetDirectory, "package.json");
+  if (!fs.existsSync(packagePath)) {
+    console.error("Could not find package.json, unable to upgrade packages");
+    process.exit(1);
+  }
   try {
-    execSync(
-      `npm install @yext/sites-components@latest @vitejs/plugin-react@4.0.4 vite@latest --prefix ${targetDirectory}`,
-      {
-        stdio: "inherit",
-      }
-    );
+    const packageJson = JSON.parse(fs.readFileSync(packagePath, "utf-8"));
+    packageJson.dependencies["@yext/sites-components"] = "^1.0.0-rc.4";
```

Comment:
Is there a way to get the latest for this one instead of hardcoding?

Reply:
I think caret is the best we can do without calling npm or pnpm
https://github.com/npm/node-semver#versions

Reply:
This package works: https://www.npmjs.com/package/latest-version